tensorflow/compiler/mlir/lite/flatbuffer_to_string.cc

```diff
@@ -141,7 +141,8 @@ int main(int argc, char** argv) {
   std::string serialized_model;
   if (tflite::ReadAndVerify(argv[1], &serialized_model)) return 1;
 #if FLATBUFFERS_LITTLEENDIAN == 0
-  tflite::FlatBufferModel::ByteSwapSerializedModel(&serialized_model);
+  if (std::string(argv[1]) == "-")
```

Comment:
Can we have some explicit comments on this?  why "-"?

Reply:
@zichuan-wei Yes, "-" means that the flatbuffer model is read from `stdin`. In this situation, the tensor content should be byte-swapped from BE to LE to ensure the output text string is the same as on LE platform.

Reply:
Hi @zichuan-wei ,

I've added a comment in `flatbuffer_to_string.cc` to explain why "-" is used. Please take a look when you have some time. Thank you very much!